eth/vm/forks/london/blocks.py

```diff
@@ -84,10 +84,10 @@ class LondonMiningHeader(rlp.Serializable, MiningHeaderAPI):
 
 
 class LondonBlockHeader(rlp.Serializable, BlockHeaderAPI):
-    fields = UNMINED_LONDON_HEADER_FIELDS + [
+    fields = UNMINED_LONDON_HEADER_FIELDS[:-1] + [
```

Comment:
@carver This is unfortunate. What's the reason for the rigid order here?

Reply:
It just seems to be the unfortunate choice of the EIP, that the base fee is always at the end of list of fields.